src/Checkout.vue

```diff
@@ -19,10 +19,18 @@ export default {
       type: String,
       required: true
     },
+    mode:{
+      type: String,
+      required: false
```

Comment:
Remove if not required, the absence of the field `required` also means that the prop is not required. Please remove.

Reply:
The old request for `SKU` doesn't use `mode`, so it's not required, but the new one won't work without it.
I'm not sure if a request for `SKU` will work properly when `mode` is present. 
By the way, I can't verify this because I can't create a new SKU product. Umm.